extensions/gadgets/base.py

```diff
@@ -42,6 +42,11 @@ class BaseGadget(object):
     height_px = 0
     # Width of the gadget in pixels.
     width_px = 0
+    # The gadget panel in which this gadget shows up in the learner view.
+    # This must match the name of a stationary gadget panel in the skin.
+    # Example: 'bottom'. In the future, this property may offer other options
+    # like 'draggable'.
+    gadget_panel = ''
```

Comment:
Do we have validation checking that the only allowable choice for this is 'bottom'? If not I think it would be nice to add it.